.github/workflows/publish.yml

```diff
@@ -28,9 +28,52 @@ jobs:
           yarn test
           yarn package
 
+      - name: Login to DockerHub
+        uses: docker/login-action@v2
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_TOKEN }}
+
+      # Build and publish Docker runner (for GitLab CI/CD, etc.)
+      - name: Publish Docker image
+        run: |
+          full_version="1.2.3"
+          full_version=${full_version//v/} # Remove 'v' prefix
+          echo "full_version=$full_version"
+          
+          main_version=$(awk -F- '{print $1}' <<< $full_version)
+          major=$(awk -F '.' '{print $1}' <<< $main_version)
+          minor=$(awk -F '.' '{print $2}' <<< $main_version)
+          patch=$(awk -F '.' '{print $3}' <<< $main_version)
+          echo "major version: ${major}"
+          echo "minor version: ${minor}"
+          echo "patch version: ${patch}"
+          
+          prerelease=$(awk -F- '{print $2}' <<< $full_version)
+          echo "prerelease version: ${prerelease}"
+          
+          if [ -z "$prerelease" ]; then
+            echo "Building Docker image with tags $major, $major.$minor, $major.$minor.$patch"
+            docker build \
+              --tag "gatlingcorp/enterprise-runner:$major" \
+              --tag "gatlingcorp/enterprise-runner:$major.$minor" \
+              --tag "gatlingcorp/enterprise-runner:$major.$minor.$patch" \
+              ./docker
```

Comment:
You could workdir instead to make that clear (and replace with just `.`)

Reply:
Done, also added a docker build step to `build.yml` to at least make sure that the build command works